package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mx-design-tokens",
-  "version": "2.0.0",
+  "version": "4.0.1",
```

Comment:
4.0.1 has already been published.

Reply:
I haven't published yet. I'll make sure to change that.